### PR TITLE
EXLM-1479: close icon not showing for any 2nd filter from feature-cards params

### PR DESCRIPTION
**Pull request by gandal-adobe**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-1479 / 1305

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://bugfix-exlm-1479-closeicon--exlm--adobe-experience-league.hlx.live/
-

### CODE REVIEW

File: blocks/browse-filters/browse-filters.js [hunk 1]
```diff
@@ -364,6 +363,7 @@ function handleCheckboxClick(block, el, options) {
         label,
         value,
       });
+      decorateIcons(block);
```
Comment (noormd08) [inline blocks/browse-filters/browse-filters.js:366]: Keeping the decorateIcons inside the checkbox change is not a proper fix. Probably this needs to be added with await and move it to end of the block.

Comment (noormd08) [reply blocks/browse-filters/browse-filters.js:366]: I tried making the changes but can't push it due to the branch name length is more.

`**async** function appendTag(block, tag) {
  const tagsContainer = block.querySelector('.browse-tags-container');
  const tagEl = htmlToElement(`
    <button class="browse-tags" value="${tag.value}">
      <span>${tag.name}</span>
      <span>: </span>
      <span>${tag.label}</span>
      <span class="icon icon-close"></span>
    </button>
  `);
  tagsContainer.append(tagEl);
  tagsProxy.push({
    name: tag.name,
    value: tag.value,
  });
  **await decorateIcons(tagsContainer);**
}`

Comment (noormd08) [reply blocks/browse-filters/browse-filters.js:366]: @gandal-adobe Can you please try the above code.

Comment (gandaltest) [reply blocks/browse-filters/browse-filters.js]: @noormd08 - I've modified accordingly. Much appreciated.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**